fonduer/parser/parser.py

```diff
@@ -288,100 +406,62 @@ def _parse_sentence(self, node, state):
             for (rgx, replace) in self.replacements:
                 text = rgx.sub(replace, text)
 
-            # Lingual Parse
-            document = state["document"]
-            for parts in self.lingual_parse(document, text):
-                (_, _, _, char_end) = split_stable_id(parts["stable_id"])
-                parts["document"] = document
-                parts["sentence_num"] = state["sentence"]["idx"]
-                abs_sentence_offset_end = (
-                    state["sentence"]["abs_offset"]
-                    + parts["char_offsets"][-1]
-                    + len(parts["words"][-1])
-                )
-                parts["stable_id"] = construct_stable_id(
-                    document,
-                    "sentence",
-                    state["sentence"]["abs_offset"],
-                    abs_sentence_offset_end,
+            # Process the Paragraph
+            stable_id = "{}::{}:{}".format(
```

Comment:
Can we use the same format for stable_id?

Reply:
What do you mean?